hack/dump/README.md

```diff
@@ -0,0 +1,140 @@
+# Aggregated Logging Utilities
+
+ ## logging-dump
+`logging-dump.sh` is a tool to gather as much information as possible from your logging cluster. In order to run the script, the user must be logged in and have a administrative permissions on project `logging`
+
+Usage:
+
+`$ oc login -u admin https://openshift.example.com:8443`
```

Comment:
I was expecting there the user to provide either a token or the password so that you can run the script remotely.